packages/date-picker/src/FilterDateRangePicker/components/DateRangeValidationMessage/DateRangeValidationMessage.tsx

```diff
@@ -36,11 +38,10 @@ export const DateRangeValidationMessage = ({
     if (status.dateEnd === status.dateStart) {
       return (
         <FieldMessage
-          id={id}
           message={
             <ul className={styles.fieldMessageList}>
-              <li>{validationMessage.dateEnd}</li>
-              <li>{validationMessage.dateStart}</li>
+              <li id={dateEndId}>{validationMessage.dateStart}</li>
+              <li id={dateStartId}>{validationMessage.dateEnd}</li>
```

Comment:
```suggestion
              <li id={dateStartId}>{validationMessage.dateStart}</li>
              <li id={dateEndId}>{validationMessage.dateEnd}</li>
```
😛

Reply:
Side note (cos I can't comment on it), it looks like you don't use `styles.fieldMessageIcon` anymore, so you can remove it from all the places in this file :)